app/assets/stylesheets/pages/_contact.scss

```diff
@@ -1,4 +1,77 @@
+// .collapsing for custom transition with bootstrap
+.collapsing {
+  -webkit-transition-duration: 3.35s;
+  transition-duration: 3.35s;
```

Comment:
Maybe a bit long, I was not sure something was gonna happen

Reply:
It's must be configured when all the data will be in .CSV... because it depends on the data size